packages/mobile/src/components/app-navigator/types.ts

```diff
@@ -29,7 +29,12 @@ export type ProfileStackParamList = BaseStackParamList & {
   AccountSettingsScreen: undefined
 }
 
-export type BottomTabsParamList = {
+export type SearchGroupParamList = BaseStackParamList & {
```

Comment:
```suggestion
export type SearchParamList = BaseStackParamList & {
```

Reply:
reason i said group btw is cause they are all in a `Stack.Group`, but sure